### DISCUSSION TITLE
Display logs

### DISCUSSION THREAD

**Discussion by bruno-luca** [yamcs/yamcs#678] [Q&A] [answered]
Hello everyone!
I was testing some feature of my yamcs project and i was wondering 3 things:
1. Is there a way to display the yamcs server log on the web page? Maybe from the admin area?
2. If yes, how can i split logs in different "streams"? Let me explain: i would like a section of the web page where i can monitor my server logs and it would be perfect if i can have a window for INFO logs and an other for ERROR logs
3. How can i tell yamcs to save the server log into a file?

**Comment by fqqb** [comment, marked as answer]
There is no built-in mechanism to view system logs through the web application. You might be able to get there by making a custom log handler (= standard java) that converts to "Yamcs Events", but you will need to be careful of not creating an infinite loop this way.

As for logging to a file, you can configure this type of stuff in a file etc/logging.properties. See this page: https://docs.yamcs.org/yamcs-server-manual/administration/logging/
